Makefile

```diff
@@ -151,16 +151,21 @@ clean-glide-cache:
 deps: prebuild-check
 	$(GLIDE_BIN) --verbose install
 
-.PHONY: generate
-## Generate GOA sources. Only necessary after clean of if changed `design` folder.
-generate: prebuild-check $(DESIGNS) $(GOAGEN_BIN) $(GO_BINDATA_ASSETFS_BIN) $(GO_BINDATA_BIN)
+app/controllers.go: $(DESIGNS) $(GOAGEN_BIN)
 	$(GOAGEN_BIN) bootstrap -d ${PACKAGE_NAME}/${DESIGN_DIR}
+
+assets/js/client.js: $(DESIGNS) $(GOAGEN_BIN)
 	$(GOAGEN_BIN) js -d ${PACKAGE_NAME}/${DESIGN_DIR} -o assets/ --noexample
-	$(GOAGEN_BIN) gen -d ${PACKAGE_NAME}/${DESIGN_DIR} --pkg-path=github.com/goadesign/gorma
```

Comment:
A yes, that got removed by purpose, but it's not correct for this PR. We're not using it. Thanks for spotting.